contracts/SignatureValidatorManager.sol

```diff
@@ -24,11 +24,18 @@ contract SignatureValidatorManager is RegistryManager, ISafeProtocolFunctionHand
     constructor(address _registry, address _initialOwner) RegistryManager(_registry, _initialOwner) {}
 
     // constants
-    // Signature selector 0xb5c726cb
-    bytes4 public constant SIGNATURE_VALIDATOR_SELECTOR = bytes4(keccak256("Account712Signature(bytes32,bytes32,bytes)"));
+    // Signature selector bytes4(keccak256("Account712Signature(bytes32,bytes32,bytes)"));
+    bytes4 public constant SIGNATURE_VALIDATOR_SELECTOR = 0xb5c726cb;
 
-    // keccak256("AccountMessage(bytes message)");
-    bytes32 private constant ACCOUNT_MSG_TYPEHASH = 0x82cac545155fcbf147f2a9013809613677ac7d65498556e6d19ce43bcbf6c284;
+    // keccak256(
+    //     "SafeTx(address to,uint256 value,bytes data,uint8 operation,uint256 safeTxGas,uint256 baseGas,uint256 gasPrice,address gasToken,address refundReceiver,uint256 nonce)"
```

Comment:
I think this is the wrong type-hash, it should match the data being signed. Specifically, I think it should be this one:

https://github.com/safe-global/safe-contracts/blob/b344bd866b99f8f51fae20ea2b98680d5219028d/contracts/handler/CompatibilityFallbackHandler.sol#L14-L15

Reply:
aah...right, I copied SafeTx type hash. Should be Safe message type hash. Thanks for pointing out quickly.